db/tasks.php

```diff
@@ -0,0 +1,39 @@
+<?php
+// This file is part of Moodle - http://moodle.org/
+//
+// Moodle is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Moodle is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+
+/**
+ * Scheduled task definitions for mod_learningmap
+ *
+ * @package    mod_learningmap
+ * @copyright  2021-2024, ISB Bayern
+ * @author     Stefan Hanauska
+ * @license    http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
+ */
+
+defined('MOODLE_INTERNAL') || die();
+
+$tasks = [
+    [
+        'classname' => 'mod_learningmap\task\fill_backlink_cache',
+        'blocking' => 1,
+        'minute' => '*',
```

Comment:
it's only the backlinks feature, so maybe just run every 5 minutes? It will only be needed in case of a cache purge. In all other cases this will just generate a lot of logging entries.